the-collab-lab/tcl-34-smart-shopping-list
# List Page

_For an example of how to fill this template out, [see this Pull Request](https://github.com/the-collab-lab/tcl-3-smart-shopping-list/pull/44)._ 

## Description

This PR adds the new layout to the List screen. Some css files were also refactored. The applied layout changes impact the other screens (`Welcome` and `AddItem`), but PR #33 applies the new layout to the Welcome screen, so changes made in this PR will be ignored after the merge, and issue #35 is being worked on the new AddItem screen layout. To maintain the standard, between the two main screens (`List` and `AddItem`), the layout changes proposed in this PR were aligned with @mailliwmogollon, which is responsible for issue #35.

Also added the React Icons library for application icons.

## Related Issue

Closes #34 

## Acceptance Criteria

- The user wants to be able to see the application from his cell phone and also from his laptop without affecting the design of the application

## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|  ✓ | :sparkles: New feature     |
|  ✓ | :hammer: Refactoring       |
|    | :100: Add tests            |
|  ✓ | :link: Update dependencies |
|    | :scroll: Docs              |

## Updates

### Before

![image](https://user-images.githubusercontent.com/32148199/142531132-2d891682-730d-4be9-95be-2e33d6b7b97c.png)

### After

![desktop](https://user-images.githubusercontent.com/32148199/142531269-4caaab33-c0ea-41de-958f-9a826cee6301.PNG)
![mobile](https://user-images.githubusercontent.com/32148199/142531285-9f9ceb46-68fb-4e6e-997d-36a5dc0fc8f7.PNG)

## Testing Steps / QA Criteria

- From your terminal, pull down this branch with `git pull list-page-layout` and check that branch out with `git checkout list-page-layout`
- `npm install` to update dependencies
- `npm start` to launch the app
- Fill in the "Shared token" field with the `isa-collection` token
- Access the list screen
